app/gui2/ydoc-server/fileFormat.ts

```diff
@@ -14,8 +14,8 @@ export type VisualizationMetadata = z.infer<typeof visualizationMetadata>
 const visualizationMetadata = z
   .object({
     show: z.boolean().default(true),
-    project: visualizationProject,
-    name: z.string(),
+    project: visualizationProject.nullable(),
```

Comment:
Is it really `nullable`, or rather `nullish`/`optional`? Not sure we should be saving nulls in the file.